README.md

```diff
@@ -2,69 +2,110 @@
-1. Run `npm login --registry=https://npm.pkg.github.com --scope=@dfe-digital` and enter your github details
-2. Run `npm install` to install node dependencies
-3. Run `bin/rails db:setup` to set up the database development and test schemas, and seed with test data
-4. Run `bundle exec rails server` to launch the app on http://localhost:3000
-5. Run `./bin/webpack-dev-server` in a separate shell for faster compilation of assets
+## Getting started
 
-## Whats included in this boilerplate?
+## Running the application
 
-- Rails 6.0 with Webpacker
-- [GOV.UK Frontend](https://github.com/alphagov/govuk-frontend)
-- RSpec
-- Dotenv (managing environment variables)
-- Travis with Heroku deployment
+The application is a very standard [Rails](https://www.rubyonrails.org/) application and no
+special steps should be required in order to get up and running.
 
-## Running specs, linter(without auto correct) and annotate models and serializers
-```
-bundle exec rake
+### Locally
+
+#### Prerequisites
+
+Ensure you have the following installed on your machine:
+
+ * Ruby 2.6.5
+ * PostgreSQL (with an account that has superuser privileges)
+ * NodeJS
+ * NPM
+
+Clone the repository:
+
+```bash
+$ git clone git@github.com:DFE-Digital/curriculum-materials.git
 ```
 
-## Running specs
+Install bundler and bundle the Ruby gems:
+
+```bash
+$ bundle install
 ```
-bundle exec rspec
+
+If your database user needs a password, set that first by editing
+`config/database.yml`.  Now run the app:
+
+```bash
+$ bundle exec rails db:setup
 ```
 
-## Linting
+### On Docker
 
-It's best to lint just your app directories and not those belonging to the framework, e.g.
+From the application root, build and start the app:
 
 ```bash
-bundle exec rubocop app config db lib spec Gemfile --format clang -a
+$ docker build . -t curriculum_materials:latest
+$ docker-compose up
+```
 
-or
+To stop the app run:
 
-bundle exec scss-lint app/webpacker/styles
+```bash
+$ docker-compose down
 ```
 
-## Deploying on GOV.UK PaaS
+## Running the tests
+
+All of the unit and integration tests are written in [RSpec](https://rspec.info/). Run them
+in the conventional manner:
+
+```bash
+$ budle exec rspec
+```
 
-### Prerequisites
+Hopefully everything should be green!
 
-- Your department, agency or team has a GOV.UK PaaS account
-- You have a personal account granted by your organisation manager
-- You have downloaded and installed the [Cloud Foundry CLI](https://github.com/cloudfoundry/cli#downloads) for your platform
+## Schema
 
-### Deploy
+![Schema diagram](docs/schema.png)
 
-1. Run `cf login -a api.london.cloud.service.gov.uk -u USERNAME`, `USERNAME` is your personal GOV.UK PaaS account email address
-2. Run `bundle package --all` to vendor ruby dependencies
-3. Run `npm install` to vendor node dependencies
-4. Run `bundle exec rails webpacker:compile` to compile assets
-5. Run `cf push` to push the app to Cloud Foundry Application Runtime
+## Lingo
```

Comment:
```suggestion
## Glossary
```

Consider moving this and contributing to `.github/CONTRIBUTING.md` 

examples:-

- https://github.com/DFE-Digital/presentation-merger/blob/master/.github/CONTRIBUTING.md
- https://github.com/vuejs/vue/blob/dev/.github/CONTRIBUTING.md
- https://github.com/facebook/jest/blob/master/CONTRIBUTING.md

Reply:
Yep, good point, will do